### PR TITLE
ESCP-1602-1603: Support file status api

### PR DESCRIPTION
**Pull request by shabtaisharon**
should be merged after #107

### CODE REVIEW

**Comment by rpmoore** [comment]
Conflict

**Comment by shabtaisharon** [comment]
> Conflict

Fixed!

File: SpectraLogic.SpectraRioBrokerClient/Calls/Jobs/GetJobFileStatusesRequest.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        #region Constructors
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="GetJobFileStatusesRequest"/> class.
+        /// </summary>
+        /// <param name="jobId">The job identifier.</param>
+        /// <param name="objectName">The object name.</param>
+        public GetJobFileStatusesRequest(Guid jobId, string objectName)
+        {
+            Contract.Requires<ArgumentNullException>(objectName != null, "objectName");
```
Comment (rpmoore) [inline SpectraLogic.SpectraRioBrokerClient/Calls/Jobs/GetJobFileStatusesRequest.cs]: Does `jobId` need the same assertion?

Comment (shabtaisharon) [reply SpectraLogic.SpectraRioBrokerClient/Calls/Jobs/GetJobFileStatusesRequest.cs]: no, since Guid cannot be null

**Pull request merged**